homeassistant/components/nibe_heatpump/const.py

```diff
@@ -10,3 +11,13 @@
 CONF_WORD_SWAP = "word_swap"
 CONF_CONNECTION_TYPE = "connection_type"
 CONF_CONNECTION_TYPE_NIBEGW = "nibegw"
+CONF_CONNECTION_TYPE_MODBUS = "modbus"
+CONF_MODBUS_URL = "modbus_url"
+CONF_MODBUS_UNIT = "modbus_unit"
+
+
+class Series(Enum):
+    """Model series of heat pumps."""
+
+    S = auto()
+    F = auto()
```

Comment:
It would be nicer if the lib provided these constants

Reply:
Yup, will move these to lib in a future lib update.